ecommerce/views_test.py

```diff
@@ -0,0 +1,77 @@
+"""
+Tests for ecommerce views
+"""
+from mock import (
+    MagicMock,
+    patch,
+)
+
+from django.core.urlresolvers import reverse
+from django.test import override_settings
+from rest_framework.status import (
+    HTTP_200_OK,
+    HTTP_400_BAD_REQUEST,
+    HTTP_403_FORBIDDEN,
+)
```

Comment:
nit: I think it's better to import the `status` module and not any single code we need